packages/app/src/app/index.html

```diff
@@ -4,6 +4,8 @@
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
+  <!-- fix cross-origin error, by allowing to run JavaScript from eval or Function constructor source -->
+  <meta http-equiv="Content-Security-Policy" content="default-src: 'self' data: mediastream: blob: filesystem: *; script-src 'self' 'unsafe-eval' 'unsafe-inline' *">
```

Comment:
@CompuIves 
@lbogdan 
Alright, I enabled any schema including `data:`, `blob:`, `meadiastream:` and `filestream:` and any domain for **default-src**.

Maybe other rules are still necessary for other types of content.
https://content-security-policy.com/